gradle/libs.versions.toml

```diff
@@ -48,6 +48,10 @@ sonarqube = "5.1.0.4882"
 turbine = "1.1.0"
 zxcvbn4j = "1.9.0"
 zxing = "3.5.3"
+junit = "4.13.2"
+junitVersion = "1.1.5"
+espressoCore = "3.5.1"
```

Comment:
Let's avoid pulling in anything we aren't going to actively use at the moment. So I think we can avoid most or all of these new dependency versions.